CHANGES.rst

```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Version 1.3.1 (released 2017-03-14):
+Version 1.3.2 (released 2017-08-07):
```

Comment:
Since dojson is not in alpha/beta we have to keep the 1.3.1 notes and insert something like this in the start:

```
Version 1.3.2 (released 2017-08-07):
------------------------------------

- Removes experimental warning from MARC21 Undo feature.